hw_a.py

```diff
@@ -0,0 +1,63 @@
+"""
+TASK 1
```

Comment:
Well done on completing Task 1

Point 6/8

-1 not implementing the reset function
-0.5 inapprapriate using of static members i.e. number_of_items and total_price
-0.5 apply discount should not return False instead it should either return full or reduced price depends on the discount.